index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <title>PORTFOLIO</title>
+</head>
+<body>
+  <header>
+    <ul class="toolbar">
+      <li class="welcome"><a href="#">Welcome</li>
```

Comment:
- [ ] You are failing your linters due to a missing `a` closing tag in this line, please fix this line by adding the required `</a>`